Bai_1/src/Person.java

```diff
@@ -0,0 +1,63 @@
+
+public class Person {
+	private int unique_id;
```

Comment:
ko đặt tên biến với style là snake_case em nhé
java theo style CamelCase em nì
```suggestion
	private int uniqueId;
```